### ISSUE TITLE
Wrong type detection on datetime object substraction

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#312]
Originally reported by: **Anonymous**

---

Date objects subtraction result timedelta object. pylint doesn't handle this situation correctly.

```
#!python
from datetime import datetime

D1 = datetime(2011, 1, 1)
D2 = datetime(2012, 1, 1)

DELTA = D2 - D1  # Return datetime.timedelta instance
print(DELTA.total_seconds())
```

This results in warning _maybe-no-member_, which is false positive:

_Instance of 'datetime' has no 'total_seconds' member (but some types could not be inferred)_

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/312

**Comment by PCManticore** [comment]
I cannot reproduce this any longer, using the latest master branch.
